### PR TITLE
Puma support

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
This includes the commit form https://github.com/theforeman/puppet-foreman/pull/677

File: manifests/service.pp [hunk 1]
```diff
@@ -18,6 +18,8 @@
   }
 
   if $apache {
+    include ::apache
```
Comment (ekohl) [inline manifests/service.pp]: This feels weird to me. I'd expect this to be set in config already.

File: spec/acceptance/foreman_journald_spec.rb [hunk 2]
```diff
@@ -91,10 +91,10 @@ class { '::foreman':
   end
 
   describe command("journalctl -u #{service_name}") do
-    its(:stdout) { is_expected.to match(%r{Redirected to https://#{host_inventory['fqdn']}/users/login}) }
+    its(:stdout) { is_expected.to match(%r{Started The Apache HTTP Server}) }
```
Comment (ekohl) [inline spec/acceptance/foreman_journald_spec.rb]: This test was intended to test that the application logs to the journal. That means what's normally in production.log is now in the journal. By changing this to only test the regular messages, it's missing the point. Why that's not actually happening is unknown to me and may be an indication of an actual bug.

Comment (ehelms) [reply spec/acceptance/foreman_journald_spec.rb]: I revereted these and they passed for me locally. I don't know what I was seeing.

File: manifests/service.pp (hunk 1 above)
Comment (ehelms) [reply manifests/service.pp]: Trying to remember why this had to be there.

**Comment by ehelms** [comment]
I don't undestand these tests @ekohl -- half the time they pass, half the time they fail. Is there maybe a start up delay I need to increase?

**Comment by ekohl** [comment]
Incorporated in https://github.com/theforeman/puppet-foreman/pull/677

**Pull request closed without merging**